### PR TITLE
Relase ethercat-sys v0.2.0

### PR DESCRIPTION
**Pull request by flosse**
Can we  release ethercat-sys v0.2.0?

### CODE REVIEW

File: ethercat-sys/Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "ethercat-sys"
 description = "Binding to the Etherlab open-source EtherCAT master"
 keywords = ["ethercat", "master", "etherlab", "binding"]
-version = "0.1.3"
+version = "0.2.0"
 authors = ["Georg Brandl <g.brandl@fz-juelich.de>"]
```
Comment (birkenfeld) [inline ethercat-sys/Cargo.toml]: wanna add yourself here?

**Review by birkenfeld — APPROVED** [review]

File: ethercat-sys/Cargo.toml (hunk 1 above)
Comment (flosse) [reply ethercat-sys/Cargo.toml]: yes, thanks :)
I also added some more metadata.

**Comment by flosse** [comment]
@birkenfeld would you mind to either publish ethercat-sys v0.2.0 or grant me an access?

**Comment by birkenfeld** [comment]
sure, what's the crates.io username to invite?

**Comment by flosse** [comment]
> sure, what's the crates.io username to invite?

flosse

**Comment by flosse** [comment]
Thanks a lot, it worked :+1: 

I'll do some more tests tomorrow before publishing both crates :)

**Pull request merged**